app/views/request/_sidebar.html.erb

```diff
@@ -15,44 +15,25 @@
     <%= render :partial => 'track/rss_feed', :locals => { :track_thing => @track_thing, :location => 'sidebar' } %>
   </div>
 
-  <% if @info_request.described_state != "attention_requested" %>
```

Comment:
Yeah that's a good point. Really it should be on `/request/:url_title/report/new` where you actually start entering your reasons.